readme.md

```diff
@@ -41,116 +54,157 @@ A common pattern in the API is a "browser object" which is just a plain object w
 
 A browser object may only have one of `browser` or `device` set; which property is set will depend on `os`.
 
-### worker objects
+#### worker objects
 
 Worker objects are extended [browser objects](#browser-objects) which contain the following additional properties:
 
 * `id`: The worker id.
 * `status`: A string representing the current status of the worker.
-  * Possible statuses: `"running"`, `"queue"`.
+* Possible statuses: `"running"`, `"queue"`.
+
+#### screenshot objects
+
+A "screenshot object" is a configuration object to run a screenshot job.
+
+It is a plain object with properties defined on [BrowserStack's Screenshot API reference](https://www.browserstack.com/screenshots/api#generate-screenshots).
 
-### BrowserStack.createClient( settings )
+
+#### job objects
+
+A "job object" is a screenshot job object, commonly used to keep track of the screenshots requested using the [screenshot object](#screenshot-objects). They may also be displayed in the web interface on BrowserStack website.
+
+It is a plain object with properties defined on [BrowserStack's Screenshot API reference](https://www.browserstack.com/screenshots/api#screenshots-states).
+
+### REST API v4
+
+*Note: For earlier versions of the API, please see [the wiki](https://github.com/scottgonzalez/node-browserstack/wiki/API).*
+
+#### BrowserStack.createClient( settings )
 
 Creates a new client instance.
 
 * `settings`: A hash of settings that apply to all requests for the new client.
-  * `username`: The username for the BrowserStack account.
-  * `password`: The password for the BrowserStack account.
-  * `version` (optional; default: `4`): Which version of the BrowserStack API to use.
-  * `server` (optional; default: `{ host: "api.browserstack.com", port: 80 }`): An object containing `host` and `port` to connect to a different BrowserStack API compatible service.
+* `username`: The username for the BrowserStack account.
+* `password`: The password for the BrowserStack account.
+* `version` (optional; default: `4`): Which version of the BrowserStack API to use.
+* `server` (optional; default: `{ host: "api.browserstack.com", port: 80 }`): An object containing `host` and `port` to connect to a different BrowserStack API compatible service.
```

Comment:
Why were these all outdented? It now looks like the method takes many parameters, as opposed to defining what properties exist for the `settings` parameter.